src/views/Dashboard.vue

```diff
@@ -38,13 +38,14 @@
 			</DashboardWidgetItem>
 		</template>
 		<template v-slot:empty-content>
-			<div class="empty-content">
-				<img class="empty-content__image" :src="emptyImage">
-				<p class="empty-content__text">
-					{{ t('mail', 'No messages found yet') }}
-				</p>
-				<a v-if="accounts.length === 0" :href="accountSetupUrl" class="button">{{ t('mail', 'Set up an account') }}</a>
-			</div>
+			<EmptyContent id="mail--empty-content" icon="icon-checkmark">
+				<template #desc>
+					{{ t('mail', 'No message found yet') }}
+				</template>
+		</EmptyContent>
+<div class="no-account icon-archive">
+			<a v-if="accounts.length === 0" :href="accountSetupUrl" class="button">{{ t('mail', 'Set up an account') }}</a>
```

Comment:
This should be part of the empty content view as well

Reply:
It will also then center the button properly

Reply:
hmm, i cannot style it while inside, thats why i removed it, but let me try again

Reply:
it works with div :)